app/scripts/modules/amazon/src/loadBalancer/configure/network/NLBListeners.tsx

```diff
@@ -25,6 +25,85 @@ export interface INLBListenersState {
   oidcConfigs: IAuthenticateOidcActionConfig[];
 }
 
+export interface INLBCertificateSelectorProps {
+  availableCertificates: IALBListenerCertificate[];
+  certificates: { [accountId: number]: IAmazonCertificate[] };
+  formik: FormikProps<IAmazonNetworkLoadBalancerUpsertCommand>;
+  app?: Application;
+  certificateTypes: string[];
+}
+
+export class CertificateSelector extends React.Component<INLBCertificateSelectorProps> {
+  public render() {
+    return <CertificateSelectorContent {...this.props} />;
+  }
+}
+
+export function CertificateSelectorContent(props: INLBCertificateSelectorProps) {
```

Comment:
```suggestion
export function CertificateSelectorContent({ availableCertificates, certificates, formik, app, certificateTypes }: INLBCertificateSelectorProps) {
```

Then you don't have to retype `props.___`  everytime.

Reply:
updated